tests/baselines/reference/jsxNamespaceImplicitImportJSXNamespaceFromPragmaPickedOverGlobalOne.errors.txt

```diff
@@ -0,0 +1,69 @@
+/index.tsx(2,28): error TS2708: Cannot use namespace 'React' as a value.

```

Comment:
I'm not sure how to fix this, nor if this is acceptable in this test case. Namespaces in TS are magic to me 😅

Reply:
Drop something like an `export class Component {}` into the global `react` ns in the test.

Reply:
Err, but actually I think this error indicates a place we're still looking at the default factory entity name, and not the import-source based one - _might_ need to debug this to see where this error is being made from, and change it over to look inside the jsx runtime namespace instead.

Reply:
<img src="https://media.giphy.com/media/51q5VAkqgl5Ze/giphy.gif"/>

Reply:
I believe I've fixed this particular issue - although not sure if the introduced condition is enough. Please re-check.

Reply:
So this error is coming from:
https://github.com/microsoft/TypeScript/blob/d72829757de7d496908e8ba52088e11f7c8613d4/src/compiler/checker.ts#L25422
and from what I understand this only used to prevent eliding jsx-related symbol from the emitted code, so I've wrapped that part of the code so it's now only executed when `!getJsxNamespaceContainerForImplicitImport(node)`. Not sure if this is enough though - please re-check.

Reply:
That looks pretty good. The emitter is responsible for adding the implicit import, so there's no need to even lookup the non-implicit one of we're not using it.